docs/src/library.md

```diff
@@ -131,6 +131,19 @@ compare_kets
 
 The [SnowflakePlots](https://github.com/anyonlabs/SnowflakePlots.jl) package provides multiple visualization tools for Snowflake.jl. Please see the documentation of [SnowflakePlots](https://github.com/anyonlabs/SnowflakePlots.jl) for more details. 
 
+## Clifford Simulator
+Snowflake provides tools for the efficient storage and manipulation of Clifford gates and
```

Comment:
We don't yet support efficient manipulation of Clifford gates. We only have efficient Pauli gates.

Reply:
I have removed the mention of Clifford gates and I have renamed the section header to `Pauli Simulator`.